pytube/extract.py

```diff
@@ -248,7 +248,8 @@ def get_ytplayer_config(html: str) -> Any:
     """
     logger.debug("finding initial function name")
     config_patterns = [
-        r";ytplayer\.config\s*=\s*({.*?});",
+        r";ytplayer\.config\s*=\s*({.*?(?<!gdpr)});",
```

Comment:
What's the context for the gdpr bit? Do you happen to have an example video where we need to capture this?

Reply:
Unfortunately it happens rarely and not reliably that the "gdpr" appears in the dictionary. It is then enclosed by two curly brackets with a following semicolon. This would lead to a premature hit and the dictionary would be invalid. Therefore this string is ignored.

Reply:
I'll take your word for it then.